src/features.md

```diff
@@ -4,42 +4,63 @@ Some functionality is gated behind Cargo features, which must be enabled as
 required, e.g.
 
 ```
-rand = { version = "0.6", features = ["log"] }
+rand = { version = "0.7", features = ["log", "serde1"] }
 ```
 
+### Small RNG
+
+From Rand version 0.7, the [`SmallRng`] type is gated behind the `small_rng`
+feature flag (this reduces `rand`'s depnedency count by one if not used).
+
 ### Logging
 
-Rand can optionally log a few events related to `OsRng` and `thread_rng`. This
-is gated behind the `log` feature.
+Rand can optionally log a few events (mostly related to reseeding of
+[`ThreadRng`] and to [`rand_jitter`]).
 
 ### Serde
 
 Some parts of the Rand lib support serialisation via [Serde](https://serde.rs/).
 To enable this, use the `serde1` feature.
 
+Note that cryptographic RNGs *do not* support serialisation since this could be
+a security risk. If you need state-restore functionality on a cryptographic RNG,
+the ChaCha generator supports [getting and setting the stream position](https://rust-random.github.io/rand/rand_chacha/struct.ChaCha20Rng.html#method.get_word_pos),
+which, together with the seed, can be used to reconstruct the generator's state.
+
 ### SIMD support
 
-SIMD support is still experimental, and is gated behind the `simd_support`
-feature.
+Experimental support for generating SIMD values is available under the
+`simd_support` feature gate. This requires nightly Rust.
 
-### Nightly-only features
+The ChaCha implementations provided by [`rand_chacha`] use SIMD operations
+internally even on stable Rust. This is enabled by default.
 
-The `nightly` feature enables some less-stable functionality. If you use this,
-don't be surprised when your build breaks!
+### No-std mode
 
-Historically, `nightly` was required for `i128` support; this is now stable.
+Almost all Rand libraries can be used without Rust's standard library:
 
-The `nightly` feature enables `simd_support`, along with some additional
-SIMD functionality not available on stable compilers.
+-   For the [`rand`] and [`rand_chacha`] crates, one must disable default features
+    (`default-features = false`) and some functionality is lost.
+-   The [`rand_core`] and [`rand_jitter`] crates assume `no_std` by default, but
+    support some additional functionality when the `std` feature is enabled.
+-   [`getrandom`] (and by extension [`rand_os`]) has limited support for `no_std`.
+    It does not use of feature flags but detects based on the target platform.
```

Comment:
This sentence is not clear to me. Maybe replace "of" with "any"? What does it detect based on the target platform?